iree/turbine/kernel/wave/templates/paged_decode_attention.py

```diff
@@ -111,7 +111,7 @@ def phase_0_constraints():
         constraints += [tkw.WorkgroupConstraint(B, BLOCK_B, 1, primary=False)]
         constraints += [tkw.WaveConstraint(B, BLOCK_B / B_WAVES)]
 
-        constraints += [tkw.WorkgroupConstraint(S, BLOCK_S, 2)]
+        constraints += [tkw.WorkgroupConstraint(S, BLOCK_S, 0)]
```

Comment:
Can you explain why we flipped the distribution dimension between S and U (0 <-> 2)

Reply:
nvm, I see you writing it helped slightly haha